ui/src/components/sidebar/SidebarDownloadMedia.tsx

```
@@ -55,7 +55,9 @@ const formatBytes = (t: TranslationFn) => (bytes: number) => {
   }
 }
 
-const downloadMedia = (t: TranslationFn) => async (url: string) => {
+const { add, removeKey } = useMessageState()
+
+const downloadMedia = (t: TranslationFn, add: Function, removeKey: Function) => async (url: string) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)